### PR TITLE
Offline: Fixed mask (as gmap) and zoom constraints for offline mode

### PR DESCRIPTION
**Pull request by fgravin**
See details in https://github.com/camptocamp/ngeo/pull/3965

### CODE REVIEW

File: geoportailv3/templates/index.html [hunk 1]
```diff
@@ -252,16 +252,14 @@ <h2 translate>Feedback</h2>
         </div>
         <div ng-if="mainCtrl.selectedLayers.length>0" class="slider-layer-label"><i class="fa fa-arrow-left"></i> <span>{{mainCtrl.selectedLayers[0].get('label') | translate}}</span></div>
       </div>
-      <ngeo-offline class="offline-debug" ng-if="::mainCtrl.debugOffline"
-          debug="::true"
+      <ngeo-offline class="offline-debug" ng-class="{'offline-debug': mainCtrl.debugOffline}"
```
Comment (ger-benjamin) [inline geoportailv3/templates/index.html]: You must remove the class (already in your ng-class)

Comment (ger-benjamin) [reply geoportailv3/templates/index.html]: You probably can use `::mainCtrl.debugOffline` in your ng-class (with `::`)

Comment (gberaudo) [reply geoportailv3/templates/index.html]: We can also get rid of the offline-debug stuff now.

Comment (fgravin) [reply geoportailv3/templates/index.html]: FYI `{'offline-debug': ::mainCtrl.debugOffline}` is not a valid syntax.

I'll remove the stuff about debug then.

**Comment by fgravin** [comment]
@ger-benjamin i've removed the offline debug in the markup and fixed the `_`

**Pull request merged**